### ISSUE TITLE
Font licenses

### ISSUE THREAD

**Issue by baldo** [leungwensen/svg-icon#40]
Hey, that's a great collection you've compiled there. One thing that's preventing me from using it currently, is that the licenses of each font aren't included. Also I wonder if you've made sure, if redistributing the icons that way is compliant with each font's license (I'm not sure if redistribution as SVGs can be considered a derivative work).

It would be great to:

- [ ] Include the licenses of each font.
- [ ] Check if there is any license preventing redistribution in such a way.
- [ ] Clarify that the MIT license applies to your code, but probably not to the fonts / icons in use (perhaps add clarification in the `LICENSE` file).

**Comment by leungwensen** [comment]
Good point! It is a serious problem, I will handle it in the coming version
